collections/_article/maintenance-release-godot-3-6-1.md

```diff
@@ -140,8 +140,8 @@ This release is built from commit [`b1ba98fce`](https://github.com/godotengine/g
 
 The downloads for this dev snapshot can be found directly on our repository:
```

Comment:
It's not a dev snapshot actually :D 

And we have a widget for downloads, see recent 4.x blog posts for the template to copy.

I didn't review the original post as I thought @Repiteo would have caught this kind of things 😅

Reply:
I honestly just assumed the widget was only compatible with 4.x, because the links only seemed to have 4.x containers. I'm probably just not familiar enough with the website overall though